packages/taxdat/R/setup_helpers.R

```diff
@@ -1457,11 +1520,14 @@ config_defaults[["processing"]][["remove_unobserved_space_slices"]] <- function(
   return(TRUE)
 }
 config_defaults[["stan"]] <- list()
+config_defaults[["stan"]][["nchain"]] <- function(config, index) {
+  return(4)
+}
 config_defaults[["stan"]][["ncores"]] <- function(config, index) {
-  return(2)
+  return(pmax(config[["stan"]][["nchain"]], parallel::detectCores()))
```

Comment:
should it be "nchain" or "ncores"? the name of the function shows it is related to ncores. Also, in old codes, it was ncores.